qcengine/compute.py

```diff
@@ -104,44 +129,39 @@ def compute_procedure(input_data: Dict[str, Any],
         A QC Schema representation of the requested output, type depends on return_dict key.
     """
 
-    # Build the model and validate
-    input_data = model_wrapper(input_data, OptimizationInput, raise_error)
-    if isinstance(input_data, FailedOperation):
-        if return_dict:
-            return input_data.dict()
-        return input_data
+    procedure = procedure.lower()
```

Comment:
I feel like we could generalize this structure in a call, but its fine for now since it only comes up twice.